var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -258,6 +259,8 @@ def configure_args(self):
 
         if "+rocm" in spec:
             rocm_pkgs = ["rocsparse", "rocthrust", "rocprim", "rocrand"]
+            if "+superlu-dist" in spec:
```

Comment:
FYI. We're now encouraging use of `self.spec.satisfies()` to check variants of the package.  As the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#testing-spec-constraints state:

`The difference between the two functions is that satisfies() tests whether spec constraints overlap at all, while in tests whether a spec or any of its dependencies satisfy the provided spec.`

Reply:
perhaps this can be a global change to hypre spec in a different PR @osborn9 ?

Reply:
Yes - I'll make those updates in a separate PR.